### PR TITLE
Added Package Endpoint provider and Package service tests

### PR DESCRIPTION
**Pull request by singhraman4282**
**What's this do?**
Tests for Package Endpoint provider and Package Service 

**Why are we doing this? (w/ JIRA link if applicable)**
N/A

**How should this be tested? / Do these changes have associated tests?**
Cannot be QA tested

**Dependencies for merging? Releasing to production?**
N/A

**Does this include changes that require a new SimplyE build for QA?**
N/A

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
Raman Singh verified it

### CODE REVIEW

File: SimplifiedTests/NYPLAxisTests/NYPLPackageServiceTests/NYPLAxisPackageServiveTests.swift [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      }
+      if self.fileExists(at: chapter3URL) {
+        writeExpectation.fulfill()
+      }
+    }
+    
+    waitForExpectations(timeout: 3, handler: nil)
+  }
+
+  private func mockDownloadedContainer() {
```
Comment (ettore) [inline SimplifiedTests/NYPLAxisTests/NYPLPackageServiceTests/NYPLAxisPackageServiveTests.swift:165]: I couldn't understand what this function was doing without looking at the code, can we rename this? Doe you think `writeAssetForContainer()` work here?

File: SimplifiedTests/NYPLAxisTests/NYPLPackageServiceTests/NYPLAxisPackageServiveTests.swift [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    }
+    itemDownloader.delegate = terminationListener
+    packageService.downloadPackageContent()
+    
+    waitForExpectations(timeout: 3, handler: nil)
+  }
+  
+  func testPackageServiceShouldNotContinueUponPackageWriteFailure() {
+    let terminationExpectation = self.expectation(
+      description: "Pacakge download should terminate if pacakge file write fails")
```
Comment (ettore) [inline SimplifiedTests/NYPLAxisTests/NYPLPackageServiceTests/NYPLAxisPackageServiveTests.swift:94]: minor, double typo :)

**Review by ettore — APPROVED** [review]

File: SimplifiedTests/NYPLAxisTests/NYPLPackageServiceTests/NYPLAxisPackageServiveTests.swift (hunk 2 above)
Comment (singhraman4282) [reply SimplifiedTests/NYPLAxisTests/NYPLPackageServiceTests/NYPLAxisPackageServiveTests.swift]: Done

File: SimplifiedTests/NYPLAxisTests/NYPLPackageServiceTests/NYPLAxisPackageServiveTests.swift (hunk 1 above)
Comment (singhraman4282) [reply SimplifiedTests/NYPLAxisTests/NYPLPackageServiceTests/NYPLAxisPackageServiveTests.swift]: Done

**Review by ErnestFan — APPROVED** [review]

**Pull request merged**